pymatgen/analysis/phase_diagram.py

```diff
@@ -88,9 +78,6 @@ def energy(self) -> float:
         """
         return self._energy
 
-    def __repr__(self):
-        return "PDEntry : {} with energy = {:.4f}".format(self.composition, self.energy)
```

Comment:
Why is this `__repr__` removed?

Reply:
Now inherited from the `Entry` baseclass using `self.__class__.__name__`